docs/source/CONTRIBUTING.md

```diff
@@ -102,15 +102,17 @@ speaking, Git handles attribution automatically.
 
 ### Setting Up A Development Environment
 
-As a general best practice, you should work on this project in an isolated
-Python virtual environment. The following set of commands will set that up for
-you:
+This project uses [Poetry] to manage its dependencies and do a lot of the heavy
+lifting. This includes managing development environments! If you are not
+familiar with this tool, we highly recommend checking out [their docs][poetry docs]
+to get used to the basic usage.
 
-```sh
-> python -m venv .venv
-> source .venv/bin/activate
-> pip install -e .[tests]
-```
+Now, setting up a development environment is super simple!
+
+Step 1: [Install Poetry]  
```

Comment:
Consider including a link on how to install poetry?
https://python-poetry.org/docs/#installation

Reply:
[Like this](https://github.com/godaddy/tartufo/pull/52/files#diff-c8ea9f02a746ef0a0c7e94b9c291a944R187)? 😄